packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -559,19 +562,21 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
                   },
                 ]
               : []),
-            ...[
-              {
-                label: 'Links',
-                item: (
-                  <div className="cards-column">
-                    <LinkedUrlCard
-                      thread={thread}
-                      allowLinking={isAuthor || isAdminOrMod}
-                    />
-                  </div>
-                ),
-              },
-            ],
+            ...(isAuthor || isAdmin || hasWebLinks
```

Comment:
should we add `isAdminOrMod` condition here? like we do below =>  `allowLinking={isAuthor || isAdminOrMod}`